.github/workflows/test_develop.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   trigger_tests:
     strategy:
       matrix:
-        branch: [develop, release/cycle-3.117, release/cycle-3.116]  # List other branches here
+        branch: [develop, release/cycle-3.118]  # List other branches here
```

Comment:
Why did you change it? I changed this script in the develop branch. I thought (according to the checklist) we need to update it in the develop branch

Reply:
I thought that way I won't forget about it later.
It really doesn't matter, since only what is on develop is considered.